src/cbmc/cbmc_parse_options.cpp

```diff
@@ -310,8 +332,14 @@ void cbmc_parse_optionst::get_command_line_options(optionst &options)
   PARSE_OPTIONS_GOTO_CHECK(cmdline, options);
 
   // generate unwinding assertions
-  if(cmdline.isset("unwinding-assertions"))
+  // TODO: Fotis: revisit
+  if(cmdline.isset("no-unwinding-assertions"))
   {
+    options.set_option("unwinding-assertions", false);
+    options.set_option("paths-symex-explore-all", false);
+  } else {
+    // Not really needed, as it's now on by default, but keeping it here
+    // for completeness' sake.
```

Comment:
Also, please avoid the word "completeness".

Reply:
Apologies, I was using the word "completeness" here not in a verification context, but more to indicate symmetry between the two branches (consequent and alternative), despite options setting in one of them being redundant (because the options have been set above), so that reading the code is easier compared to leaving the option setting in the context be implicit.

I will update the wording to make this clearer.